### PR TITLE
Chris candidates modal

### PR DESCRIPTION
**Pull request by ryalia44**

### CODE REVIEW

File: src/components/CandidateModal/index.jsx [hunk 1]
```diff
@@ -0,0 +1,145 @@
+import React from 'react';
+import { CMS_BASE_URL } from 'constants/api';
+
+const CandidateModal = ({ candidate }) => {
+  const getDesiredDetail = key => {
+    let desiredDetail;
+    candidate.details.map(detail => {
+      if (detail.title === key) {
+        desiredDetail = detail;
+        return '';
```
Comment (AmyHong0502) [inline src/components/CandidateModal/index.jsx]: I know I wrote it but we don't need this line. Could you remove line 10?

File: src/components/CandidateModal/index.jsx [hunk 2]
```diff
@@ -0,0 +1,145 @@
...
+
+  const displayContact = contactMethodList.map(cmItem => {
+    let contactMethod = ``;
+    let key = `${cmItem}`;
+    const found = candidate.contacts.find(
+      contact => contact.contactMethod === cmItem
+    );
+    if (found) {
+      if (found.contactMethod === 'Instagram') {
+        let splitURL = found.contactValue.split('/');
```
Comment (AmyHong0502) [inline src/components/CandidateModal/index.jsx]: Please use `const` when you're not going to change the value.

File: src/components/CandidateModal/index.jsx [hunk 3]
```diff
@@ -0,0 +1,145 @@
...
+    'Other'
+  ];
+
+  const displayContact = contactMethodList.map(cmItem => {
+    let contactMethod = ``;
+    let key = `${cmItem}`;
+    const found = candidate.contacts.find(
+      contact => contact.contactMethod === cmItem
+    );
+    if (found) {
```
Comment (AmyHong0502) [inline src/components/CandidateModal/index.jsx]: Can we use a `switch` statement? I think it'll be easier to understand.

Comment (christruong90) [reply src/components/CandidateModal/index.jsx]: unable to resolve this at this moment

**Pull request merged**